rdt-open-reader/src/covidIndonesia/assets/json.form/patient-diagnostics-form-original.json

```diff
@@ -88,22 +88,18 @@
           "err": "This field is required"
         }
       },
-      {
-        "key": "facility_name_label",
-        "type": "label",
-        "text": "Name of facility",
-        "text_color": "#000000"
-      },
       {
         "key": "facility_name",
         "openmrs_entity_parent": "",
         "openmrs_entity": "",
         "openmrs_entity_id": "",
-        "type": "edit_text",
+        "type": "spinner",
+        "hint": "Name of facility",
         "v_required": {
           "value": "false",
           "err": ""
-        }
+        },
+        "values": []
```

Comment:
The spinner here and in all other instances should follow the [new spinner format](https://github.com/OpenSRP/opensrp-client-native-form/wiki/Spinner-Field#spinner-20-mls-20-compatible) that uses an options block.

Reply:
fixed.

Reply:
I can't find the fix for this. Did you push it?

Reply:
I have fixed that locally but didn't push the changes